docs/presentation/Customization/SortingRule.md

```diff
@@ -15,32 +15,110 @@ instances are first sorted by the highest priority rule and then the lower prior
-
-### Example
-
-```JSON
-{
-  "ruleType": "PropertySorting",
-  "requiredSchemas": [{ "name": "MySchema", "minVersion": "1.2.3" }],
-  "priority": 999,
-  "class": { "schemaName": "MySchema", "className": "MyClass" },
-  "isPolymorphic": true,
-  "propertyName": "MyProperty"
-}
+| Name                                            | Required? | Type                                                                 | Default                        |
+| ----------------------------------------------- | --------- | -------------------------------------------------------------------- | ------------------------------ |
+| *Filtering*                                     |
+| [`requiredSchemas`](#attribute-requiredschemas) | No        | [`RequiredSchemaSpecification[]`](../Advanced/SchemaRequirements.md) | `[]`                           |
+| [`priority`](#attribute-priority)               | No        | `number`                                                             | `1000`                         |
+| [`condition`](#attribute-condition)             | No        | [ECExpression](../hierarchies/ECExpressions.md#rule-condition)       | `""`                           |
+| [`class`](#attribute-class)                     | No        | `SingleSchemaClassSpecification`                                     | All classes in current context |
+| [`isPolymorphic`](#attribute-ispolymorphic)     | No        | `boolean`                                                            | `false`                        |
+| *Sorting*                                       |
+| [`propertyName`](#attribute-propertyname)       | Yes       | `string`                                                             |                                |
+| [`sortAscending`](#attribute-sortascending)     | No        | `boolean`                                                            | `true`                         |
+
+### Attribute: `requiredSchemas`
+
+> **Default value:** `[]`
+
+A list of ECSchema requirements that need to be met for the rule to be used. See more details in [Defining ECSchema Requirements for Presentation Rules](../Advanced/SchemaRequirements.md).
+
+```ts
+[[include:Sorting.RequiredSchemas.Ruleset]]
 ```
 
+| `minVersion` value   | Result                                                                 |
+| -------------------- | ---------------------------------------------------------------------- |
+| `minVersion = 1.0.2` | ![Example of using required schemas condition](./media/sortingrule-sorted-asc.png) |
+| `minVersion = 2.0.2` | ![Example of using required schemas condition](./media/sortingrule-unsorted.png)   |
+
+### Attribute: `priority`
+
+> **Default value:** `1000`
+
+Defines the order in which rules are handled - higher priority means the rule is handled first. If priorities are equal, the rules are handled in the order they're defined.
```

Comment:
This description is taken from `ContentRule` documentation page. I have omitted `onlyIfNotHandled` attribute from this description and this documentation page because it has no effect on `SortingRule`. 

It can be added to documentation when rules engine starts using `onlyIfNotHandled` attribute and it has an effect on `SortingRule`